al2/x86_64/standard/3.0/Dockerfile

```
@@ -226,6 +229,26 @@ RUN set -x \
     # Install Amazon Corretto 11
     # Note: We will use update-alternatives to make sure JDK11 has higher priority for all the tools
     && yum install -yq java-11-amazon-corretto \
+    && for tool_path in $JAVA_HOME/bin/*; do \
+          tool=`basename $tool_path`; \
+          update-alternatives --install /usr/bin/$tool $tool $tool_path 10000; \
+          update-alternatives --set $tool $tool_path; \
+        done
+
+RUN set -x \
+    # Install Amazon Corretto 17
+    # Note: We will use update-alternatives to make sure JDK11 has higher priority for all the tools
+    && export GNUPGHOME="$(mktemp -d)" \
+    && curl -fL -o corretto.key https://yum.corretto.aws/corretto.key \
+    && gpg --batch --import corretto.key \
+    && gpg --batch --export --armor '6DC3636DAE534049C8B94623A122542AB04F24E3' > corretto.key \
+    && rpm --import corretto.key \
+    && rm -r "$GNUPGHOME" corretto.key \
+    && curl -fL -o /etc/yum.repos.d/corretto.repo https://yum.corretto.aws/corretto.repo \
+    && grep -q '^gpgcheck=1' /etc/yum.repos.d/corretto.repo \
+    && yum install -y java-17-amazon-corretto-devel-17.0.0.35-1 \
```

While I think pinning versions in Docker images is useful and should be encouraged, the other versions of Amazon Corretto are not pinned (lines 208 & 231).

Should we "unpin" Amazon Corretto 17 here?
```suggestion
    && yum install -y java-17-amazon-corretto-devel \
```

---

Thanks for that. I will check for other potential occurrences of that.